sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/util/QueryChargeHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.azure.digitaltwins.core.util;
+
+import com.azure.core.http.rest.PagedResponse;
+
+import java.util.Objects;
+
+/**
+ * A helper to extract the query charge from the query response.
+ * An Azure Digital Twins Query Unit (QU) is a unit of on-demand computation that's used to execute your Azure Digital Twins queries.
+ */
+public final class QueryChargeHelper {
+    private static final String queryChargeHeader = "query-charge";
+
+    public static <T> Float getQueryCharge(PagedResponse<T> page) {
```

Comment:
should we return a primitive type `float` instead, with value 0? Or do you think that might be misleading?

Reply:
I felt 0 would be misleading compared to null so did not use the primitive type.